model/clusters_mgmt/v1/aws_sts_account_roles_inquiry_resource.model

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages STS Roles
+resource AWSSTSAccountRolesInquiry {
+    // Retrieves the list of STS Account Roles in the specified AWS Account.
+    // IMPORTANT: This collection doesn't currently support paging or searching, so the returned
+    // `page` will always be 1 and `size` and `total` will always be the total number of available account roles
+    // in the AWS Account
+
+    method Search {
+
+        // AWS Account Details required for the inquiry
+        in Body AWS
+
+        // Index of the returned page, where one corresponds to the first page. As this
+        // collection doesn't support paging the result will always be `1`.
+        in out Page Integer = 1
+
+        // Number of items that will be contained in the returned page. As this collection
+        // doesn't support paging or searching the result will always be the total number of
+        // be the total number of STS account roles.
+        in out Size Integer = 100
+
+        // Total number of items of the collection that match the search criteria,
+        // regardless of the size of the page. As this collection doesn't support paging or
+        // searching the result will always be the total number of STS account roles
+        out Total Integer
```

Comment:
Looking at the implementation there is neither `total`, nor `size` nor `page` being returned. And for the later two, we don't make use of them for now.  I suppose we could drop them from here for now to reduce the API surface.

Reply:
@machi1990 This is just the API change first. I have an additional backend change that will add support for those fields 👍

Reply:
Understood. Thanks for the reply!